runtime/vm/exceptiondescribe.c

```diff
@@ -34,6 +34,7 @@
 #include "vm_internal.h"
 #include "jvminit.h"
 #include "SCQueryFunctions.h"
+#include "../shared_common/shrinit.h"
```

Comment:
This can be avoided if you put the new structs `RomToRamEntry`, `RomToRamQueryEntry` and `ROM_TO_RAM_QUERY_TAG` into `SCQueryFunctions.h`